src/sentry/static/sentry/app/views/acceptProjectTransfer.tsx

```diff
@@ -7,9 +8,21 @@ import NarrowLayout from 'app/components/narrowLayout';
 import SelectField from 'app/views/settings/components/forms/selectField';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 import {t, tct} from 'app/locale';
+import {Organization, Project} from 'app/types';
 
-class AcceptProjectTransfer extends AsyncView {
-  getEndpoints() {
+type Props = RouteComponentProps<{}, {}>;
+
+type TransferDetails = {
+  organizations: Organization[];
+  project: Project;
+};
+
+type State = {
+  transferDetails: TransferDetails;
+} & AsyncView['state'];
+
+class AcceptProjectTransfer extends AsyncView<Props, State> {
+  getEndpoints(): [string, string, any][] {
```

Comment:
seems like some other refactors end up using `ReturnType<AsyncView['getEndpoints']>` instead of `[string, string, any][]` I guess this is more of a question for others too, but which one is usually preferred?

Reply:
I prefer `ReturnType<AsyncView['getEndpoints']> `😉

Reply:
I'm on the `ReturnType` bandwagon as well.